index.bs

```diff
@@ -10067,7 +10068,7 @@ in the node.
 		* {{[[callable process]]}} is `true`.
 
 		* The {{AudioWorkletNode}} is [=actively processing=], meaning
-			ANY of the following states are met:
+			ANY of the following conditions are met:
```

Comment:
FYI: https://github.com/WebAudio/web-audio-api/pull/1875/files

Reply:
Yeah, well, I was only reading the current spec, not considering things PRs that might change things. :-)

Reply:
We can merge this first, and then I can handle the conflict in #1875.